itou/approvals/models.py

```diff
@@ -601,11 +612,25 @@ def next_min_start_at(approval):
         """
         Returns the minimum date on which a suspension can begin.
         """
+        # by default the next min start is the date of last hiring start of user
```

Comment:
```suggestion
        # Default starting date.
```
Cela étant, je pense pas que ces commentaires soient toujours pertinents maintenant que le code est plus clair.